tests/tensorflow_autolog/test_tensorflow2_autolog.py

```diff
@@ -700,14 +723,6 @@ def test_tf_estimator_autolog_model_can_load_from_artifact(tf_estimator_random_d
     mlflow.tensorflow.load_model("runs:/" + tf_estimator_random_data_run.info.run_id + "/model")
 
 
-@pytest.mark.large
```

Comment:
This test is no longer necessary since autologging config updates are implemented & tested via higher-level `autologging_utils` APIs. Removing it. This was failing due to line 708 after the changes to always log metrics for the last epoch were implemented.